src/conn/conn_layered.c

```diff
@@ -1103,6 +1103,19 @@ __wti_disagg_conn_config(WT_SESSION_IMPL *session, const char **cfg, bool reconf
         WT_ERR(__wt_config_gets(session, cfg, "disaggregated.lose_all_my_data", &cval));
         if (cval.val != 0)
             F_SET(&conn->disaggregated_storage, WT_DISAGG_NO_SYNC);
+
+        /*
+         * Get the percentage of a page size that a delta must be less than in order to write that
+         * delta (instead of just giving up and writing the full page).
+         */
+        WT_ERR(__wt_config_gets(session, cfg, "disaggregated.delta_pct", &cval));
```

Comment:
Should we create a new category for page delta configurations instead of putting them within the disaggregated config? E.g. `page_delta.delta_pct`? .

Reply:
agree with this. We should not tie delta to disaggregated only

Reply:
@mm-ng I have a suggestion: if we're moving page delta configurations, it would be logical to consolidate all five related configs (delta_pct, max_consecutive_delta, internal_page_delta, leaf_page_delta, flatten_leaf_page_delta) into a new 'page_delta' category. I'll keep the current ticket's acceptance criteria (btree to connection level migration) as is and create a separate ticket for this broader category refactoring. What do you think?

Reply:
will that be a lot of work? I think the change is acceptable in the same pr.

Reply:
Fixed in 9045dd9bec9427cebb92e4c555eb30e86fd75b23